src/solver/simulation/adequacy.cpp

```diff
@@ -68,15 +68,7 @@ bool Adequacy::simulationBegin()
         pProblemesHebdo.resize(pNbMaxPerformedYearsInParallel);
         for (uint numSpace = 0; numSpace < pNbMaxPerformedYearsInParallel; numSpace++)
         {
-            SIM_InitialisationProblemeHebdo(study, pProblemesHebdo[numSpace], 168, numSpace);
```

Comment:
Here we used to give value 168 as argument, and then check its value (through problem hebdo).
- What the point in using a hard coded value end then check its value
- Furthermore, **nbHoursInAWeek** is existing, is used in this source file. So why not use it ?

We use **nbHoursInAWeek** instead of 168 and remove the useless check.